src/main/resources/templates/pages/dashboard/org/index.peb

```diff
@@ -0,0 +1,58 @@
+{# @pebvariable name="model" type="org.koil.dashboard.org.OrgIndexViewModel" #}
+    {% extends "layouts/dashboard-layout" %}
+    {% block title %} {{ model.getOrganizationName }} | OrgView {% endblock %}
+
+{% block content %}
+<section>
+    <hgroup>
+        <h2 data-test="organization-name">{{ model.getOrganizationName }}</h2>
+        <h3 data-test="organization-signup-link">Signup Code: {{ model.getOrganizationSignupLink }}</h3>
+    </hgroup>
+    <h3>All User Accounts</h3>
+    <table class="table is-striped">
+        <thead>
+        <tr>
+            <th><a href="{{ model.sortBy("fullName") }}">Full Name {{ model.sortDirection("fullName") }}</a></th>
+            <th class="is-hidden-mobile"><a
+                    href="{{ model.sortBy("isOrganizationOwner") }}">Owner {{ model.sortDirection("isOrganizationOwner") }}</a></th>
+            <th class="is-hidden-mobile"><a
+                    href="{{ model.sortBy("emailAddress") }}">Email {{ model.sortDirection("emailAddress") }}</a></th>
+            <th></th>
+        </tr>
+        </thead>
+        <tbody>
+                {% for account in model.accounts %}
+            <tr data-test="account-row-{{ account.emailAddress }}">
+                <td>{{ account.fullName }}</td>
+                <td class="is-hidden-mobile">{{ account.isOrganizationOwner }}</td>
+                <td class="is-hidden-mobile">{{ account.emailAddress }}</td>
+                <td><a href="/dashboard/org/accounts/{{ account.accountId }}" data-test="user-details">Details</a></td>
+            </tr>
+                {% endfor %}
+        </tbody>
+    </table>
+    <div class="grid is-justify-centered">
+        <div>
+            Results per page:
+            <a href="?size=20">20</a>
+            <a href="?size=50">50</a>
+            <a href="?size=100">100</a>
+            <a href="?size=500">500</a>
+        </div>
+        <div>
+            {% if model.hasPrevious() %}
```

Comment:
I thought that I had extracted this as a component?

Reply:
I don't see a component like this anywhere so I don't think so.

Reply:
I'll create a todo for this.